backend/src/agents/intent_agent.py

```diff
@@ -11,7 +11,7 @@
     tools=[],
 )
 class IntentAgent(Agent):
-    def invoke(self, utterance: str) -> str:
+    async def invoke(self, utterance: str) -> str:
         user_prompt = engine.load_prompt("intent", question=utterance)
 
-        return self.llm.chat(self.model, intent_format, user_prompt=user_prompt, return_json=True)
+        return await self.llm.chat(self.model, intent_format, user_prompt=user_prompt)

```

Comment:
So we don't want the return_json = True anymore?

Reply:
good catch, I had to resolve a lot of merge conflicts and must have made a mistake here. Will add return_json=True back in

Reply:
Have pushed this change now :)